### PR TITLE
Panels: Keybindings

### PR DESCRIPTION
**Pull request by gtk-grafana**
Fixes: https://github.com/grafana/explore-logs/issues/945
Also fixes a bug in the fields breakdown where the explore query was going to the logs panel query instead of the time series query for the panel. Added e2e coverage for this case since it was missed, and as a result a bug was released.

### Shortcuts:

**Focused panel options**
`p x`: Panel Explore
`p l`: toggle legend visibility

**Time range options**
`d r`: Refresh time range
`t z`: Zoom out
`ctrl+z`: Zoom out
`t a`: Switch to absolute time range
`t ◀️ `: shift time range left
`t ▶️ `: shift time range right

NOT USED IN GRAFANA CORE:
`a l`: toggle legend visibility for all panels in view (This was more of a demonstration of what we can do, not sure if we want this or not)

Follow ups:
* Save panel options to local storage
* Add these options (which ones) to PanelMenu?
* More options?

### CODE REVIEW

**Comment by matyax** [comment]
How can users discover these keybindings if they don't know about them?

File: src/services/Mousetrap.ts [hunk 1]
```diff
@@ -0,0 +1,998 @@
+/**
```
[the remark this answers is not in the document]
Comment (gtk-grafana) [reply src/services/Mousetrap.ts:1]: This is how it was included in [core Grafana](https://github.com/grafana/grafana/blob/main/public/app/core/services/mousetrap/Mousetrap.ts)

https://github.com/grafana/grafana/pull/88603

Looks like so we could add typescript support? https://github.com/grafana/grafana/pull/88603/commits/ef30941f1dfa04c3d0facdadf16c7a7eb3c31acd

**Comment by gtk-grafana** [comment]
> How can users discover these keybindings if they don't know about them?

Consistency with core Grafana for now, but we should follow up by adding things to the PanelMenus and maybe add a cheat-sheet like in core. 
image

**Comment by matyax** [comment]
Cool!

**Review by matyax — APPROVED** [review]

**Comment by gtk-grafana** [comment]
@svennergr when you're back in office maybe we want to add a keyboard shortcut for the investigations links? I was going to add it to this PR, but wasn't sure how to gate it. Would we check the `appSidecar` feature flag?

**Pull request merged**